File: lua/blink/delimiters/langs/cpp.lua
--- @type blink.delimiters.LanguageDefinition
return {
  delimiters = {
    ['('] = ')',
    ['['] = ']',
    ['{'] = '}',
  },
  line_comment = { '//' },
  block_comment = { '/*', '*/' },
  string = { '"', "'" },
  -- TODO: handle R$prefix block strings
  block_string = {},
}

File: src/delimiters/languages/cpp.rs
use crate::define_token_enum;

define_token_enum!(CppToken, {
    delimiters: {
        "(" => ")",
        "[" => "]",
        "{" => "}"
    },
    line_comment: ["//"],
    block_comment: ["/*" => "*/"],
    string: ["\"", "'"],
    block_string: []
});
